src/PAModel/Serializers/MsAppSerializer.cs

```diff
@@ -566,7 +566,7 @@ private static IEnumerable<FileEntry> GetMsAppFiles(this CanvasDocument app, Err
 
             RepairComponentInstanceIndex(app._entropy?.ComponentIndexes ?? new Dictionary<string, double>(), sourceFiles);
 
-            foreach (var sourceFile in sourceFiles)
+            foreach (var sourceFile in sourceFiles.OrderBy(file => file.GetMsAppFilename()))
```

Comment:
Can we add a a comment stating this is important (so that it doesn't get reverted)